wordpress-mobile/WordPress-iOS
# Implement Read it to me feature.

Use AVSpeechSynthesizer to read posts.

<img src="https://user-images.githubusercontent.com/651601/77375937-dec89300-6d66-11ea-9429-0c2d22b0cc11.gif" width=300>

To test:
 - Open the WP app
 - Open a Reader post
 - Tap on the play icon on the toolbar
 - Check if post is show and correctly read/speak by the device.

PR submission checklist:
